scripts/aggregate_cells.py

```diff
@@ -0,0 +1,67 @@
+import os
+import faulthandler
+import pandas as pd
+from pycytominer.cyto_utils.cells import SingleCells
+
+
+def aggregate(sql_file, metadata_dir, barcode_platemap):
+    """_summary_
+
+    Args:
+        sql_file (str): _description_
+        metadata_dir (str): _description_
+        barcode_platemap (str): _description_
+    """
+
+    # Loading appropriate platemaps with given plate data
+    plate = os.path.basename(sql_file).rsplit(".", 1)
+    barcode_platemap_df = pd.read_csv(barcode_platemap)
+    platemap = barcode_platemap_df.query(
+        "Assay_Plate_Barcode == @plate"
+    ).Plate_Map_Name.values[0]
+    platemap_file = os.path.join(metadata_dir, "platemap", "{}.csv".format(platemap))
+    platemap_df = pd.read_csv(platemap_file)
+
+    # TODO: generate a configuration file that contains the default parameters
```

Comment:
We have this now right? Is it time to update this script using the config file?

Reply:
Yes that what i am working on right now. Those notes were just there to let me know which processes needs to be in our config file. 

removed: 21305ee51b64193213e31d112de1f6e5e17eff03